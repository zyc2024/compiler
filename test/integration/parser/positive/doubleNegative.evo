int main() {
    int k = -2 - 2;
    int j = -2 - - 2;
    int l = -2 - -9223372036854775808;
    return 0;
}