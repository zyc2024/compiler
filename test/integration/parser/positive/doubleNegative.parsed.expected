(
    ()
    (
        (main () (int) (
            (= (k int ) ( - (- 2) 2) )
            (= (j int ) (- (- 2) (- 2)))
            (= (l int ) (- (- 2) -9223372036854775808 ))
            (return 0)
        ))
    )
)